common/djangoapps/student/tests/test_activate_account.py

```diff
@@ -103,6 +104,10 @@ def test_account_activation_message(self):
         response = self.client.get(reverse('dashboard'))
         self.assertNotContains(response, expected_message)
 
+    def _assert_user_active_state(self, expected_active_state):
```

Comment:
**Note to Reviewers:** I updated these tests in order to merge test functionality that was in `AccountActivationAndPasswordChangeTest` in `openedx/core/djangoapps/user_api/accounts/tests/test_api.py`.